homeassistant/components/mill/climate.py

```diff
@@ -1,5 +1,10 @@
 """Support for mill wifi-enabled home heaters."""
-from mill import Mill
+from __future__ import annotations
+
+from collections.abc import Sequence
+from typing import Any, Callable
+
+from mill import Heater, Mill
```

Comment:
You're an author of `mill` library, why not you add type hints there first? It looks pretty small.

Reply:
Out of scope for this PR, but should we enable `warn_return_any` in MyPy?

Reply:
Actually it should be already enabled. I'll check why it doesn't work in this case.

Reply:
> Shows a warning when returning a value with type `Any` from a function declared with a non- `Any` return type.

https://mypy.readthedocs.io/en/stable/config_file.html#confval-warn_return_any

Here underlying library is untyped, therefore all arguments have `Any` type.